### PR TITLE
Forward them IPs

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: src/schema.ts [hunk 1]
```diff
@@ -112,6 +112,12 @@ const makeSchema = async () => {
       authorization: `Bearer ${previousContext.graphqlContext &&
         previousContext.graphqlContext.getToken &&
         previousContext.graphqlContext.getToken()}`,
+      ...(previousContext.graphqlContext &&
+        previousContext.graphqlContext.headers &&
+        previousContext.graphqlContext.headers && {
```
Comment (palmenhq) [inline src/schema.ts:117]: i think we have one `previousContext.graphqlContext.headers` too much or?

File: src/schema.ts [hunk 2]
```diff
@@ -112,6 +112,12 @@ const makeSchema = async () => {
       authorization: `Bearer ${previousContext.graphqlContext &&
         previousContext.graphqlContext.getToken &&
         previousContext.graphqlContext.getToken()}`,
+      ...(previousContext.graphqlContext &&
```
Comment (palmenhq) [inline src/schema.ts:115]: what happens if we try spreading `undefined`? i think this maybe should be 
```
<the has headers check> ? {
  'x-forwarded-for': ...,
} : {}
```

its not pretty but it's darn safe

File: src/schema.ts [hunk 3]
```diff
@@ -112,6 +112,12 @@ const makeSchema = async () => {
       authorization: `Bearer ${previousContext.graphqlContext &&
```
Comment (palmenhq) [inline src/schema.ts:112]: two lines up, shall we also rename this from `authorizationContextLink`?

File: src/schema.ts (hunk 2 above)
Comment (hedvigoscar) [reply src/schema.ts:115]: spreading undefined is valid and results in nothing:

```
> {foo:'bar', ...(undefined)}
{ foo: 'bar' }
```

Comment (hedvigoscar) [reply src/schema.ts:115]: This actually holds true for most falsy values:
```
> {foo:'bar',...(false)}
{ foo: 'bar' }
> {foo:'bar',...(null)}
{ foo: 'bar' }
```

File: src/schema.ts (hunk 3 above)
Comment (hedvigoscar) [reply src/schema.ts:112]: `authorizationAndIpLink`? Honestly I think the name is still kind of fine 🤔

Comment (palmenhq) [reply src/schema.ts:112]: still think its confusing haha, maybe it should be two separate links rather?

Comment (hedvigoscar) [reply src/schema.ts:112]: It is admittedly a hack. Two links might be the way to go. Having this functionality for all links would probably be desirable, so maybe the IP-one can serve as a base-link that the others join upon

**Pull request merged**